playbooks/roles/kibana-email-alert/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+# what to match in journald log
+kibana_email_alert_regex: "Server log"
```

Comment:
Updated with more specific regex - ``"\"type\":\"log\".+\"tags\".+\"actions\".+Server log:"`` 
- Should be the type of log (there are others like request, response)
- There should be tags and ``actions`` after tags
- And should have the text ``Server log``. This text is the same for any Server Log connector. The connector name doesn't show up, otherwise, we could've used that.